packages/apps/purchase/src/screens/RequestListScreen.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useMemo, useState} from 'react';
+import {ChipSelect, Screen} from '@axelor/aos-mobile-ui';
+import {
+  SearchListView,
+  useSelector,
+  useTranslator,
+  useTypeHelpers,
+  useTypes,
+} from '@axelor/aos-mobile-core';
+import {RequestCard, SupplierSearchBar} from '../components';
+import {searchPurchaseRequest} from '../features/purchaseRequestSlice';
+
+const displayPurchaseRequestSeq = item => item.purchaseRequestSeq;
+
+const RequestListScreen = ({}) => {
+  const I18n = useTranslator();
+  const {PurchaseRequest} = useTypes();
+  const {getSelectionItems} = useTypeHelpers();
+
+  const {
+    loadingPurchaseRequests,
+    moreLoadingPurchaseRequest,
+    isListEndPurchaseRequest,
+    purchaseRequestList,
+  } = useSelector(state => state.purchase_purchaseRequest);
+
+  const [selectedStatus, setSelectedStatus] = useState([]);
+  const [supplier, setSupplier] = useState();
+
+  const statusList = useMemo(() => {
+    const statusToDisplay = [
+      PurchaseRequest?.statusSelect.Draft,
+      PurchaseRequest?.statusSelect.Requested,
+      PurchaseRequest?.statusSelect.Accepted,
+    ];
```

Comment:
I think we should either show all statuses in the ChipSelect or only display the concerned statuses in the list.
Currently, it's not really consistent. Let's check with MCH when he comes back.